totalRP3_Extended_Tools/aura/effects.lua

```diff
@@ -0,0 +1,462 @@
+local EMPTY = TRP3_API.globals.empty;
+local getClass = TRP3_API.extended.getClass;
+local setTooltipForSameFrame = TRP3_API.ui.tooltip.setTooltipForSameFrame;
+local registerEffectEditor = TRP3_API.extended.tools.registerEffectEditor;
+local registerOperandEditor = TRP3_API.extended.tools.registerOperandEditor;
+local loc = TRP3_API.loc;
+local stEtN = TRP3_API.utils.str.emptyToNil;
+
+local function getAuraNameFromClassId(classId)
+	local id = tostring(classId);
```

Comment:
`classId` could be `nil`, which would cause the `getClass` call to crash.
I took that pattern from https://github.com/Total-RP/Total-RP-3-Extended/blob/df5faeb970383b2851bb71043acaa910c10d6faa/totalRP3_Extended_Tools/items/effects.lua#L104

Reply:
Right, fair enough on that one. I think the other ones cannot be nil though, as they all have `or ...` in the save function, so they should be safe to remove.